examples/just_enhancement.py

```diff
@@ -120,7 +120,6 @@ def run_scans(nScans, power_idx, DNP_data=None):
         print("\nRUNNING BOARD...\n")
         this_time_var = time.time()
         SpinCore_pp.runBoard()
-        power_start_times.append(this_time_var)
```

Comment:
I didn't like the scope of this -- list is created outside the function, at the top, but here you're manipulating it like a global variable.